BaseballMitt.cpp

```diff
@@ -5,10 +5,22 @@ BaseballMitt::BaseballMitt(std::string a_color) :
     color(std::move(a_color))
 {}
 
-void BaseballMitt::print_type() const {
-    std::cout << "BaseballMitt Type is " << this->type << std::endl;
+int BaseballMitt::get_type() const {
+    return type;
 }
 
 void BaseballMitt::set_type(int type) {
     this->type = type;
 }
+
+const std::string & BaseballMitt::get_color() const {
+    return color;
+}
+
+void BaseballMitt::set_color(std::string color) {
+    this->color = std::move(color);
+}
+
+void BaseballMitt::print_maker() {
+    std::cout << "Mitt maker is Wilson" << std::endl;
+}
```

Comment:
This is unusual, too.
It's inflexible -- what if the user of your class wants the output to go somewhere else? To a file, or to cerr, or to a stringstream?
As you found earlier today, it's hard to test, too.
Better would be to have a get_maker function:
```
const char *get_maker() const {
  return "Wilson";
}
```

Reply:
I agree